client/src/pages/Scheduler.js

```diff
@@ -54,9 +54,11 @@ const Scheduler = () => {
 
   useEffect(() => {
     // Matching only the first two params so /hostname/eventname/datelinks will still work correctly
-    const regex = /appt\/([0-9a-z-]*\/[0-9a-z-]*)/g;
+    const regex = /appt\/([%\s0-9a-z-]*\/[%\s0-9a-z-]*)/g;
```

Comment:
I wonder if we should just grab the string and run a `decodeURIComponent()` on it. But if this works then let's just go with it.

Reply:
This works but that could possibly be a better way!